### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by Aman9das** [Arcticons-Team/Arcticons#355] [Ideas]
Widget suggestions thread

Feel free to ask for widgets to be added..

**Comment by Aman9das** [comment]
![Screenshot_20210914_203941.jpg](https://user-images.githubusercontent.com/39594914/133283970-e8a9d243-683f-4461-a9ac-0f8dbbe13b08.jpg)

Arcticons isn't viable for widgets rn..
But alright.. date widget coming soon!

**Reply by Aman9das** [reply]
What is the date format you want @Donnnno 

rn its "Tue, 14 Sep"

**Reply by Aman9das** [reply]
![Screenshot_20210914_223247.jpg](https://user-images.githubusercontent.com/39594914/133302067-85b6aabd-fdc1-4f56-9093-b9a0745d5cb9.jpg)

![Screenshot_20210914_223602.jpg](https://user-images.githubusercontent.com/39594914/133302317-756bfce7-0e2e-48f6-93c8-99c437098517.jpg)

Pushing these changes.. about screen picture and date widget @Donnnno

**Comment by mubashir-rehman** [comment]
- Can we get Now Playing(feature in Pixel and many other launchers) widget?
-  Also, widget should follow the lines color.
- weather widget
- location widget
Many of them can be considered out of scope but as you find it useful
